hub/powertoys/run.md

```diff
@@ -190,7 +195,7 @@ The PowerToys Run calculator plugin supports the following operations:
 
 
 ### Time and Date values
-The time and date plugin provides the current time and date or a custom one in different formats. You can enter the format or a custom time/date or both when searching.
+The Time and Date plugin provides the current time and date or a custom one in different formats. You can enter the format or a custom time/date or both when searching.
```

Comment:
This should be one plug-in, you sure both words need a Capital? The words in the [note] below should follow.

Reply:
my mistake. good catch. Thinking if we should write plugin names italic or bold to mark it some way? 🤔

Reply:
On this page we have explicit headers, so therefore I would say no need.